package/network/utils/comgt/files/ncm.sh

```diff
@@ -84,6 +84,7 @@ proto_ncm_setup() {
 	[ -n "$delay" ] && sleep "$delay"
 
 	manufacturer=`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | awk '/Manufacturer/ { print tolower($2) }'`
+	manufacturer=`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | head -2 | tail -1 | tr -d "\r" | tr [A-Z] [a-z]`
```

Comment:
I just sent a related patch to the mailing list, where among other changes the AT+CGMI command is used for identifying the modem. I think it should be straightforward to use that.